python/aiffairness/setup.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import setup, find_packages
+
+tests_require = [
+    'pytest',
+    'pytest-tornasync',
+    'mypy'
+]
+
+setup(
+    name='aifserver',
+    version='0.1.1',
+    author_email='Andrew.Butler@ibm.com',
+    license='https://github.com/kubeflow/kfserving/LICENSE',
+    url='https://github.com/kubeflow/kfserving/python/aifserver',
+    description='Model Server implementation for AI fairness. \
+                 Not intended for use outside KFServing Frameworks Images',
+    long_description=open('README.md').read(),
+    python_requires='>3.4',
+    packages=find_packages("aifserver"),
+    install_requires=[
+        "kfserving>=0.2.1",
```

Comment:
Should the latest release be put here (0.5.1)? Seems like other setup.py files were updated in this regard.